src/chain-wallet-libs/bindings/wallet-js/tests/certificates/vote_cast.rs

```diff
@@ -0,0 +1,68 @@
+use crate::{generate_settings, generate_wallet};
+use wallet_js::*;
+use wasm_bindgen_test::*;
+
+#[wasm_bindgen_test]
+fn vote_cast_public_test() {
+    let mut wallet = generate_wallet().unwrap();
+
+    let settings = generate_settings().unwrap();
+
+    let vote_plan = VotePlanId::from_bytes(&[
+        0, 1, 2, 3, 4, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25,
+        26, 27, 28, 29, 30, 31, 32,
+    ])
```

Comment:
We can use `let bytes: [u8; 32] = core::array::from_fn(|i| i);` to generate this slice

Reply:
fixed